### DISCUSSION TITLE
Music CD-ripping without metadata from the internet possible?

### DISCUSSION THREAD

**Discussion by debian-stormwatch** [whipper-team/whipper#546] [Q&A] [answered]
Hi to you all!

When ripping a music-CD where no CD-metadata can be downloaded from the internet, Whipper stops the ripping-process. Is there a way, to force Whipper, so that it rips CD's without that data?

Many thanks for any help!

Peter

**Comment by JoeLametta** [comment, marked as answer]
Hi!
Yes, it's possible (whipper even works, with reduced features, without a network connection) but you need to provide the `--unknown` (or just `-U`) option when running `whipper cd rip`. For more details, please check the [relevant man page](https://github.com/whipper-team/whipper/blob/develop/man/whipper-cd-rip.rst).

**Comment by debian-stormwatch** [comment]
Hi Joe! Thanks a million. That's great news!!!
Cheers, Peter
